spec/Cabin/Infotainment.vspec

```diff
@@ -211,3 +211,20 @@ SmartphoneProjection.SupportedMode:
   type: attribute
   allowed: [ 'ANDROID_AUTO', 'APPLE_CARPLAY', 'MIRROR_LINK', 'OTHER' ]
   description: Supportable list for projection.
+
+PowerOptimize:
+  type: branch
+  description: All PowerOptimize actions.
+
+PowerOptimize.DisplayBrightnessLevel:
```

Comment:
I see two points that needs to be discussed

**Intended logic**

If this signal is added we will have two brightness signals, this one and `HMI.Brightness`. How is an HMI/Display supposed to behave? Shall it use `HMI.Brightness` normally but `PowerOptimize.DisplayBrightnessLevel` if PowerOptimize mode is set? If so, then I think it would make sense to add the "global" PowerOptimize signal first so it is clear when this signal shall be used.

**Naming**

We already have a signal `HMI.Brightness`, this signal use the term `DisplayBrightnessLevel`. Assuming that they refer to the same display I think we should align the names, for example by making sure they both have the same leaf-name, e.g. either  `HMI.Brightness` and  `PowerOptimize.Brightness`, or `HMI.DisplayBrightnessLevel` and  `PowerOptimize.DisplayBrightnessLevel` or some other aligned naming alternative,

Reply:
Not followed all the discussion, but are we expecting Poweroptimize sub-branches in other areas as well? Would that not be a better fit for an overlay?